features/step_definitions/user_view_steps.rb

```diff
@@ -0,0 +1,9 @@
+# Given /^I am on the user information page for "(.*)"$/ do |username|
+#     u = User.find_by_name(username)
+#     visit user_path(u)
+# end
+
+# Then /^I should be on the user information page for "(.*)"$/ do |username|
+#     u = User.find_by_name(username)
+#     expect(visit user_path(u))
+# end
```

Comment:
This file's steps seem to be commented out. Is this file necessary or were you planning to use these somewhere?

Reply:
oh they have to be removed. Thanks for catching!